index.html

```diff
@@ -196,17 +227,21 @@ <h2>Tonic</h2>
     </section>
 <section >
     <!-- modify this form HTML and place wherever you want your form -->
-    <form action="https://formspree.io/f/mwkdaqzw" method="POST" id="contact1">
-        <h1>Contact me</h1>
-        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs
-        coding.
-        I’d love to help with it!</p>
-                <input type="text" class="userName" placeholder="lorem Ipsum" maxlength="30" required>
-                <input type="email" class="email" id="#" placeholder="youremail@gmial.com" required>
-                <textarea class="report" id="#" cols="30" rows="10" maxlength="500" placeholder="Write your message 500 max words" required></textarea>
-        <!-- your other form fields go here -->
-        <button type="submit">Get in touch</button>
-    </form>
+        <form action="/" method="POST" id="contact1">
+            <h1>Contact me</h1>
+            <p>If you have an application you are interested in developing, a feature that you need built, or a project that
+                needs coding. I’d love to help with it!</p>
+            <input type="text" class="userName" placeholder="lorem Ipsum" maxlength="30" id="name" required>
+            <span class="error-message" id="name-error"></span>
+            <input type="email" class="email" id="email" placeholder="youremail@gmail.com" required>
+            <small class="error-message" id="email-error"></small>
+            <textarea class="report" id="textarea" cols="30" rows="10" maxlength="500"
+                placeholder="Write your message (500 max words)" required></textarea>
+            <!-- your other form fields go here -->
+            <button type="submit">Get in touch</button>
+        </form>
```

Comment:
- [ ] Your form action need to add formspree url to work properly and can change the placeholder for name and email.
- [ ] Form validation is working but when I sent the message show like that 

![Screenshot (201)](https://github.com/OrdinaryMack/Portfolio-Contact-form/assets/118582316/4fa2f02e-0a07-4676-b876-0dedcad85c21)